exercises/concept/need-for-speed/.docs/hints.md

```diff
@@ -28,11 +28,10 @@
 
 ## 6. Check if a remote control car can finish a race
 
-- Solving this is probably best done by [repeatedly driving the car][while].
+- Try applying a formula that compares the distance and speed against the battery and battery drain.
 - Remember that the car has a method to retrieve the distance it has driven.
 - Consider what to do when the battery has been drained before reaching the finish line.
```

Comment:
I think the last hint doesn't make sense for the formula approach and is kinda covered by the new first hint. I'd suggest removing it for the formula approach.

Reply:
Oh yeah, you're right!